configs/base.yml

```diff
@@ -0,0 +1,39 @@
+environment: &env
```

Comment:
Does compose handle this ok ? I thought it might need to be a key on an actual service? Great if it does.